config/crd/bases/datasciencecluster.opendatahub.io_datascienceclusters.yaml

```diff
@@ -443,6 +443,16 @@ spec:
                         - Removed
                         pattern: ^(Managed|Unmanaged|Force|Removed)$
                         type: string
+                      registriesNamespace:
+                        default: odh-model-registries
+                        description: Namespace for model registries to be installed,
+                          configurable once, defaults to "odh-model-registries"
+                        type: string
+                        x-kubernetes-validations:
+                        - message: RegistriesNamespace is immutable
+                          rule: self == oldSelf
+                    required:
```

Comment:
Should this be a required field? Any addition of required field will need DSC api version update.

Can we follow same cadence as other fields , where the fields are optional with default values set

Reply:
The primary requirement is that Dashboard MUST have a value in this property so they know which model registry namespace to use. So, it can't be an optional/empty value. I looked, but we don't seem to have a Default() implementation to provide 
If the new required property has a default value, k8s seems to add it automatically when the CRD is updated. So, is an api version update still needed?

Reply:
I did find the common webhook in `admission/webhook.go`, but we aren't using it to set any default values it seems.

Reply:
Getting more inputs here https://github.com/opendatahub-io/opendatahub-operator/pull/1221#discussion_r1750485311